_vector-search/ai-search/hybrid-search/collapse.md

```diff
@@ -0,0 +1,252 @@
+---
+layout: default
+title: Using collapse with a hybrid query
+parent: Hybrid search
+grand_parent: AI search
+has_children: false
+nav_order: 10
+---
+
+# Using collapse with hybrid query
+**Introduced 3.1**
+{: .label .label-purple }
+
+You can collapse on a field by specifying the `collapse` parameter in the search query.
+This will return the highest scoring document for each unique value of that field, up to the specified size in the query.
+The collapse parameter requires the field being collapsed to be of either a `keyword` or a `numeric` type.
+
+Note: Inner hits compatibility within the collapse description is not currently supported for hybrid query.
+
+The following example is taken from the documentation for [collapse in non-hybrid query](https://docs.opensearch.org/docs/latest/search-plugins/collapse-search/) and modified for hybrid query:
+
+
+Create an index:
+```json
+PUT /bakery-items
+{
+  "mappings": {
+    "properties": {
+      "item": {
+        "type": "keyword"
+      },
+      "category": {
+        "type": "keyword"
+      },
+      "price": {
+        "type": "float"
+      },
+      "baked_date": {
+        "type": "date"
+      }
+    }
+  }
+}
+```
+
+Index documents:
+```json
+POST /bakery-items/_bulk
+{ "index": {} }
+{ "item": "Chocolate Cake", "category": "cakes", "price": 15, "baked_date": "2023-07-01T00:00:00Z" }
+{ "index": {} }
+{ "item": "Chocolate Cake", "category": "cakes", "price": 18, "baked_date": "2023-07-04T00:00:00Z" }
+{ "index": {} }
+{ "item": "Vanilla Cake", "category": "cakes", "price": 12, "baked_date": "2023-07-02T00:00:00Z" }
```

Comment:
Can you add 2-3 items of `Vanilla Cake` and then show it got collapsed to just 1 the response?